Typescript_assignment/js/page4.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+class Bank_Acc {
+    constructor(id, firstName, lastName, address, phone, email, type) {
+        this.collection = [];
```

Comment:
I think this.collection=[] should not be in constructor ...this would clean your array every time and then you will have only one element in an array